build.sh

```diff
@@ -195,6 +195,9 @@ launch_terminal() {
                 # If we couldn't determine the actual terminal, try the most compatible approach
                 "$terminal" -e bash -c "$modified_cmd" &
                 ;;
+            "alacritty")
+                "$terminal" --title="$title $instance_id" -e bash -c "$modified_cmd" &
```

Comment:
The title should be `--title="AeroSim Unreal Instance $instance_id"` since `$title` is not used in this script.